core/capabilities/ccip/launcher/deployment.go

```diff
@@ -1,142 +1,45 @@
 package launcher
 
 import (
-	"fmt"
+	"golang.org/x/exp/maps"
 
 	cctypes "github.com/smartcontractkit/chainlink/v2/core/capabilities/ccip/types"
-
+	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
 	"go.uber.org/multierr"
-
-	ccipreaderpkg "github.com/smartcontractkit/chainlink-ccip/pkg/reader"
 )
 
-// activeCandidateDeployment represents a active-candidate deployment of OCR instances.
-type activeCandidateDeployment struct {
-	// active is the active OCR instance.
-	// active must always be present.
-	active cctypes.CCIPOracle
-
-	// candidate is the candidate OCR instance.
-	// candidate may or may not be present.
-	// candidate must never be present if active is not present.
-	candidate cctypes.CCIPOracle
-}
-
-// ccipDeployment represents active-candidate deployments of both commit and exec
-// OCR instances.
-type ccipDeployment struct {
-	commit activeCandidateDeployment
-	exec   activeCandidateDeployment
-}
-
-// Close shuts down all OCR instances in the deployment.
-func (c *ccipDeployment) Close() error {
-	// we potentially run into this situation when
-	// trying to close an active instance that doesn't exist
-	// this check protects us from nil pointer exception
+type ccipPlugins map[ocrtypes.ConfigDigest]*cctypes.CCIPOracle
```

Comment:
```suggestion
type ccipPlugins map[ocrtypes.ConfigDigest]cctypes.CCIPOracle
```

Pointer to an interface type is unnecessary 99% of the time in Go